firefox-ios/nimbus.fml.yaml

```diff
@@ -13,20 +13,24 @@ channels:
   - beta
   - release
 include:
+  - nimbus-features/accountSettingsReduxFeature.yaml
   - nimbus-features/addressAutofillFeature.yaml
   - nimbus-features/backForwardListCoordinatorRefactorFeature.yaml
   - nimbus-features/contextualHintFeature.yaml
+  - nimbus-features/credentialAutofillCoordinatorRefactor.yaml
```

Comment:
How come all these extra things are getting added to the include list? Could we double check that these should be in there already? What's `main` looking like?

Reply:
@adudenamedruby indeed you are right, I checked main and the added features are not there. I think there is some problem with the tracked files from git, because I pulled on main before making this pr.

Reply:
ok. I took a look as well. Looks like someone forgot to run a command for this. I'mma take care of that in a separate PR, just to keep your work clean. We'll get this merged in soon!

Reply:
Actually, your PR has uncovered a bug in some of our Nimbus stuff! amazing! Ty for the help!!!! hahaha

Reply:
@adudenamedruby haha nice to know. Perfect than I'll rebase once the fix is on main thank you.